Vish/assignment-6/VishnuA6_1.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Document</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="box">
+            <h2>MI-9 Intern HTML Assesment</h2>
+        </div>
+        <hr>
+        <form action="#">
```

Comment:
I don't Think we need to define action attribute if there is no submit button 
.

Reply:
ok